### PR TITLE
Menu Split and Fixes

### PR DESCRIPTION
**Pull request by Litorom**
#205 fixed

### CODE REVIEW

**Comment by Litorom** [comment]
Sorry it glitched

File: src/main/java/minicraft/screen/OptionsMainMenuDisplay.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package minicraft.screen;
```
Comment (Makkkkus) [inline src/main/java/minicraft/screen/OptionsMainMenuDisplay.java:1]: Why does OptionsDisplay still exist?

**Review by Makkkkus — CHANGES_REQUESTED** [review]
Changes are good, but you need to remove OptionsDisplay so we don't have three settings menu classes.

This is a nice little pull request, though.

File: src/main/java/minicraft/screen/OptionsMainMenuDisplay.java (hunk 1 above)
Comment (Litorom) [reply src/main/java/minicraft/screen/OptionsMainMenuDisplay.java:1]: I said in #205, that I kept it just in case.

**Review by Makkkkus — COMMENTED** [review]
Changes are good; will test after school is done

**Review by Makkkkus — APPROVED** [review]
Approved (it didn't remove the review last time)

**Comment by Litorom** [comment]
MERGE!!

**Pull request merged**